modules/examples/mlops-stepfunctions/scripts/input.yaml

```diff
@@ -0,0 +1,187 @@
+app_id: aiops
```

Comment:
Does this make sense to mount this config via [dataFiles](https://seed-farmer.readthedocs.io/en/latest/manifests.html#a-word-about-datafiles)?

Reply:
Updated with the dataFiles.

Reply:
Reverting back on this. Given the training scrips are part of the package, we gain very limited flexibility in using data files for the step function. I think it makes sense to keep it inside the module. Please revert.

Reply:
Reverted.